src/container/srv_target.c

```diff
@@ -87,8 +87,22 @@ cont_child_alloc_ref(void *key, unsigned int ksize, void *varg,
 
 	uuid_copy(cont->sc_uuid, key);
 	cont->sc_dtx_resync_time = crt_hlc_get();
-
+	cont->sc_snapshots_ts = crt_hlc_get();
+	cont->sc_aggregated_ts	= 0;
+	cont->sc_aggregation_max = 0;
+	cont->sc_snapshots_nr = 0;
+	cont->sc_snapshots = NULL;
+	cont->sc_vos_aggregating = 1;
 	*link = &cont->sc_list;
+
+	ds_cont_child_get(cont);
+	rc = dss_ult_create(ds_cont_aggregate_ult, cont,
+			    DSS_ULT_AGGREGATE, DSS_TGT_SELF, 0, NULL);
+	if (rc) {
+		D_WARN(DF_UUID": Failed to create aggregation ULT; rc %d\n",
+		       DP_UUID(cont->sc_uuid), rc);
+		ds_cont_child_put(cont);
+	}
```

Comment:
I personally don't like to bind the aggregation ULT with container cache logic. I think we may have a per VOS xstream main aggregation ULT periodically scan container table and spawn per VOS container ULT to do container aggregation (where to open and hold container).

But I'm ok to land with current approach, because it requires less code changes.